superdesk/users/users.py

```diff
@@ -15,112 +15,113 @@
 
 
 class UsersResource(Resource):
+    readonly = True
 
-    def __init__(self, endpoint_name, app, service, endpoint_schema=None):
-        self.readonly = True if app.config.get('LDAP_SERVER', None) else False
```

Comment:
@mdhaman I would have kept the above verification but I'm getting "working outside context" error, since I let go the `__init__` method. Any ideas of how I could keep the `if`?